packages/@okta/vuepress-site/docs/guides/configure-access-policy/overview/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Overview
+---
+
+## What is an access policy?
+
+Access policies help you secure your APIs by defining different access and refresh token lifetimes for a given combination of grant type, user, and scope. You create policy rules that allow code to evaluate whether an application should be permitted to access specific information from your protected APIs and for how long. Access policies are specific to a particular authorization server and the client applications that you designate for the policy.
+
+For example, an access token for a banking API may include a `transactions:read` scope with a multi-hour token lifetime. By contrast, the lifetime of an access token for transferring funds should be only a matter of minutes.
+
+You can also [include custom claims](/docs/guides/customize-authz-server/create-claims/) in ID and access tokens to disclose the information that you want to share, depending on the client and the scope of the tokens. Scopes specify what access privileges are being requested as part of the authorization. Claims are statements about the subject, such as name, role, or email address.
```

Comment:
The term "subject" might be be understood by a novice reader. I'd suggest explaining it with something like:

> Claims are statements about the user (or "subject") such as name, role, or email address.